AppController/djinn.rb

```diff
@@ -5593,8 +5529,8 @@ def try_to_scale_up(app_name, delta_appservers)
       @cluster_stats.each { |node|
         next if node['private_ip'] != host
 
-	    # Convert total memory to MB
-        total = node['memory']['total']/1024/1024
+	      # Convert total memory to MB
```

Comment:
Comment doesn't seems to line up: perhaps there is a tab? Sentence should finish with a period.